tests/trello/gui/trello_gui_board.robot

```diff
@@ -14,7 +14,10 @@ create a board
     Click New Board Button
     Set Name to Board                     NewName
     Verify Name of Board                     NewName
-    [Teardown]    Close Browser
+    Open Show Menu
```

Comment:
If "Open Show Menu" is needed for the Teardown, then it should be inside the Teardown... otherwise the Teardown will fail for failing tests...
(same for all other tests)